### DISCUSSION TITLE
"Permanent cache" feature?

### DISCUSSION THREAD

**Discussion by JetpackJackson** [navidrome/navidrome#3442] [Ideas]
Hi, I was wondering if a "permanent cache" feature could be added as an option in the dashboard. DSub2000 has a permanent cache where you can select artists that will be cached and not removed if you clear the cache to make room. This is a feature that I really like so I can make sure I have a specific subset of music cached on my phone. However it's implemented, just a sort of "cache this music to this device" would be cool to have, whether it's device specific ("sync X to connected client Foo") or just by artist or album ("sync X to connected clients") is fine by me.

**Comment by deluan** [comment]
If you are talking about the Navidrome UI working as a PWA in your phone, it does not support offline playback, as there are some serious constraints on what PWA can do. If you need this feature, you better off using a proper Subsonic client.

**Reply by JetpackJackson** [reply]
I mean I want a way (doesn't matter how) where I can set certain music to be permanently cached on my phone, that isn't tied to a client. I'm not asking for playback in the web interface, just some sort of way to cache that isn't client dependent.

**Reply by deluan** [reply]
Ah, I see. Well, you need _some_ app on your phone to do the download. You can use DSub and let its cached music "visible" to other apps:
![Screenshot 2024-11-04 at 12 50 18 PM](https://github.com/user-attachments/assets/bef0a822-1950-4b1a-ae59-2afe7913a3bf)

**Reply by JetpackJackson** [reply]
Gotcha, thanks. I'll try that out. I might try seeing about installing a client on the computer I have navidrome on, cache the music there, and then sync that to my phone through something like syncthing. I'll mark this as solved.
